code/modules/mob/living/carbon/alien/larva/larva.dm

```diff
@@ -38,6 +38,9 @@
 		amount_grown = min(amount_grown + 1, max_grown)
 	..(amount)
 
+/mob/living/carbon/alien/larva/start_pulling(atom/movable/AM, force = move_force, supress_message = FALSE)

```

Comment:
I'm doubting this should be included in this PR.

Reply:
My bad. Forgot to change it back, it was a part of the refactor on TG and it just got tossed in with everything else.